pages/linux/postfix.md

```diff
@@ -2,7 +2,7 @@
 
 > Postfix mail transfer agent (MTA) control program.
 > See also `dovecot`, a mail delivery agent (MDA) that integrates with Postfix.
-> More information: <http://postfix.org>.
+> More information: <https://www.postfix.org>.
```

Comment:
This link doesn't work for me, though I suspect this is a transitory issue.

Reply:
That's strange. @bl-ue suggested this just 2 days ago (https://github.com/tldr-pages/tldr/pull/5724#discussion_r610958600) and I'm sure the link worked. But now it doesn't work for me either.

Reply:
Yeah, it doesn't work for me either 🤷🏻 `http://` does tho: http://www.postfix.org/

Reply:
I'll change it back then